Riku/src/Riku.cpp

```diff
@@ -380,12 +368,15 @@ int main() {
 	Shader lightCubeShader("../shaders/light-vertex.shader","../shaders/light-fragment.shader");
 
 	//load used models
-	tileTypeModels.emplace_back("models/floor/water.obj",1,1);
-	tileTypeModels.emplace_back("models/floor/grass.obj",1,1);
-	tileTypeModels.emplace_back("models/floor/sand.obj",1,1);
-	Model grid_model("models/floor/grid.obj",1,1);
-	unitTypeModels.emplace_back("models/sara/model/sara.blend",1,-1);
-	unitTypeModels.emplace_back("models/Assassin/Assassin.blend",1,-1);
+	//grounds
+	front::groundModels.insert(std::make_pair("grass",Model("models/grounds/grass.obj",1,1)));
+	front::groundModels.insert(std::make_pair("sand",Model("models/grounds/sand.obj",1,1)));
+	front::groundModels.insert(std::make_pair("stone",Model("models/grounds/stone.obj",1,1)));
+	front::groundModels.insert(std::make_pair("wet",Model("models/grounds/water.obj",1,1)));
+	//biomes
+	front::biomeModels.insert(std::make_pair("forest",Model("models/biomes/forest.blend",1,1)));
+	//units
+	front::unitModel = Model("models/units/sara/model/sara.blend",1,-1);
```

Comment:
W przyszłości zrobi się klasę FrontedAssets i przeniesie się ten kod tam, ale na razie chyba nie ma na to czasu, więc niech zostanie